doc/version_history.rst

```diff
@@ -6,6 +6,20 @@
 Version History
 ###############
 
+v0.11.0
+=======
+
+* Update for ts_xml 7 and ts_idl 2.2:
+
+    * Rename SAL component and ts_idl enum module ``Rotator`` to ``MTRotator``.
```

Comment:
Not sure if this is relevant for this project but shouldn't renaming Hexapod to MTHexapod be mentioned here as well?

Reply:
I don't think so. ts_hexrotcomm uses the Rotator SAL component (and enums) for tests, but it does not use MTHexapod.